pkgs/servers/web-apps/anuko/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "anuko";
+  version = "1.22.10.5776";
+
+  # Project commits directly into master and has no release schedule.
+  # Fortunately the current version is defined in the 'APP_VERSION' constant in
+  # /initialize.php so we use its value and set the rev to the commit sha of when the
+  # constant was last updated.
+  src = fetchFromGitHub {
+    owner = "anuko";
+    repo = "timetracker";
+    rev = "fefff70c58b0bedc9f6fbe1359b92a35fcd9ec3e";
+    sha256 = "sha256-vMJmfplbkX0qqArgLaPXdwoLDsAcJ1HiFafNx0Xnmyo=";
+  };
+
+  # There's nothing to build.
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir $out
+    cp -R * $out/
+  '';
+
+  meta = with lib; {
+    description = "Simple, easy to use, open source time tracking system";
+    homepage = "https://github.com/anuko/timetracker/";
+    license = licenses.sspl;
+    maintainers = with maintainers; [ michaelshmitty ];
```

Comment:
The with pattern is discouraged, you should try to avoid it and do something like the suggestion bellow, you can read more about it here https://github.com/NixOS/nixpkgs/pull/209325
``` suggestion
meta = {
    description = "Simple, easy to use, open source time tracking system";
    homepage = "https://github.com/anuko/timetracker/";
    license = lib.licenses.sspl;
    maintainers = with lib.maintainers; [ michaelshmitty ];
```

Reply:
@Janik-Haag I'll fix it, thanks for the feedback! I'm using existing packages in nixpkgs as a base, but it's hard to know what I'm copying over is good to begin with :).
If there are more places where these kind of discouragements are documented, please let me know!

Reply:
> If there are more places where these kind of discouragements are documented, please let me know!

@Janik-Haag nevermind, I just found this https://nix.dev/anti-patterns/language